platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigratable.java

```diff
@@ -134,13 +150,29 @@ public void doVersionUpgradeImport(ImportMigrationContext context) {
 
     adminContext
         .memoryEntries()
-        .filter(
-            s ->
-                CONTENT_DIRECTORY_MONITOR_FPID.equals(s.getFactoryPid())
-                    || s.getPid().equals(URL_RESOURCE_READER_PID))
+        .filter(s -> filterConfigurationAdminEntries(s))
         .forEach(ImportMigrationConfigurationAdminEntry::restore);
   }
 
+  private boolean filterConfigurationAdminEntries(ImportMigrationConfigurationAdminEntry entry) {
+    if (entry.getPid().equals("org.codice.ddf.security.filter.login.Session")) {
+      entry.setPid("ddf.security.http.impl.HttpSessionFactory");
+    }
```

Comment:
✏️ We should do this "conversion" logic separately and after the filtering by pid/factoryPid.